litestar/typing.py

```diff
@@ -205,6 +206,9 @@ class FieldDefinition:
     name: str
     """Field name."""
 
+    def __deepcopy__(self, memo: dict[str, Any]) -> Self:
```

Comment:
What was the issue that necessitated this change?

Reply:
See the copy on controller. The listener logic with the replacement of the signature did not work with deep copy. This caused issues for owner registration.